tagstudio/src/qt/widgets/video_player.py

```diff
@@ -29,38 +30,37 @@
     QBitmap,
 )
 from PySide6.QtSvgWidgets import QSvgWidget
-from PIL import Image
 from src.qt.helpers.file_opener import FileOpenerHelper
-
-from src.core.constants import VIDEO_TYPES, AUDIO_TYPES
 from PIL import Image, ImageDraw
 from src.core.enums import SettingItems
+from src.qt.resource_manager import ResourceManager
 
 if typing.TYPE_CHECKING:
     from src.qt.ts_qt import QtDriver
 
 
 class VideoPlayer(QGraphicsView):
-    """A simple video player for the TagStudio application."""
+    """A basic video player."""
 
-    resolution = QSize(1280, 720)
-    hover_fix_timer = QTimer()
+    rm: ResourceManager = ResourceManager()
```

Comment:
I'd initialize this class inside `QtDriver` instead of here, as this will be needed in other places in the codebase as well, and that way you'll have the same instance available everywhere. Since `driver` is passed here, then it would be used as `self.driver.rm.play_icon` instead of `VideoPlayer.rm.play_icon`